src/OrbitCaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -212,7 +211,10 @@ void GpuQueueSubmissionProcessor::DeleteSavedGpuJob(int32_t thread_id,
   if (!tid_to_submission_time_to_gpu_job_.contains(thread_id)) {
     return;
   }
-  auto& submission_time_to_gpu_job = tid_to_submission_time_to_gpu_job_.at(thread_id);
+  // This method might be also called if the the last Gpu Job that was was not part of the capture,
```

Comment:
```suggestion
  // This method might be also called if the last Gpu Job that was not part of the capture,
```

Reply:
This sentence also seems incomplete? Maybe the "that" shouldn't be there?

Reply:
Yep, I'm a bit confused by this sentence.

Reply:
You are right. That sentence is completely broken. I've re-written it.